.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,28 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
We should check this is accurate once we get that information.

Reply:
Can be updated later. This is just for an initial setup.